src/db/mysql/models.rs

```diff
@@ -47,6 +48,8 @@ pub const USER_ID: &str = "userid";
 pub const MODIFIED: &str = "modified";
 pub const EXPIRY: &str = "ttl";
 pub const LAST_MODIFIED: &str = "last_modified";
+pub const COUNT: &str = "count";
+pub const TOTAL_BYTES: &str = "total_bytes";
```

Comment:
is "total_bytes" accurate? Since it's a STRING length

Reply:
Ah, thanks. It should use `TOTAL_BYTES()`